lib/zendesk_apps_tools/common.rb

```diff
@@ -10,6 +41,10 @@ def say_error(msg)
     end
 
     def get_value_from_stdin(prompt, opts = {})
+      if options[:unattended]
```

Comment:
I wonder if `--no-input` or `--without-input` might be slightly more descriptive of the option?